docs/guide/simulation-server.md

```diff
@@ -111,7 +104,7 @@ ln -s /home/cyberbotics/webots/resources/web/server/async_process.py
   "debug": true
```

Comment:
Indentation looks wrong here.

Reply:
Seems like an issue of the github viewer: https://cyberbotics.com/doc/guide/simulation-server?version=enhancement-externalize-server